### DISCUSSION TITLE
why property 'get'/'set' method can't override independ

### DISCUSSION THREAD

**Discussion by WhiteSmoogy** [dotnet/csharplang#7412] [General] [answered]
``` csharp
abstract class Cplusplus
{
	public abstract int Version {get;}
}

class CSharp :Cplusplus
{
public int Version {override get;set;}
}
``` 
compier error:
Compilation error (line 10, col 39): The modifier 'override' is not valid for this item
Compilation error (line 10, col 43): 'CSharp.Version.set': cannot override because 'Cplusplus.Version' does not have an overridable set accessor

but

``` csharp
public interface Java
{
	int Version {get;}
}

class JavaScript :Java
{
public int Version {get;set;}
}
```
compiled success.

**Comment by TahirAhmadov** [comment, marked as answer]
https://github.com/dotnet/csharplang/issues/1568
